src/js/components/covid19/awardSpendingAgency/AwardSpendingAgency.jsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * AwardSpendingAgency.jsx
+ * Created by James Lee 6/18/20
+ */
+
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { fetchAwardCount, fetchAwardAmounts, fetchAgencyCount } from 'helpers/disasterHelper';
+import DateNote from 'components/covid19/DateNote';
+import { awardSpendingAgencyTableTabs } from 'dataMapping/covid19/awardSpendingAgency/awardSpendingAgencyTableTabs';
+import MoreOptionsTabs from '../../sharedComponents/moreOptionsTabs/MoreOptionsTabs';
+import OverviewData from '../OverviewData';
+
+const overviewData = [
+    {
+        type: 'agencyCount',
+        label: 'Number of Agencies'
+    },
+    {
+        type: 'awardOutlays',
+        label: 'Award Outlays',
+        dollarAmount: true
+    },
+    {
+        type: 'awardObligations',
+        label: 'Award Obligations',
+        dollarAmount: true
+    },
+    {
+        type: 'numberOfAwards',
+        label: 'Number of Awards'
+    }
+];
+
+const AwardSpendingAgency = () => {
+    const [awardCount, setAwardCount] = useState(null);
+    const [awardOutlays, setAwardOutlays] = useState(null);
+    const [awardObligations, setAwardObligations] = useState(null);
+    const [numberOfAwards, setNumberOfAwards] = useState(null);
+    const [activeTabLabel, setActiveTabLabel] = useState(awardSpendingAgencyTableTabs[0].label);
+    const defCodes = useSelector((state) => state.covid19.defCodes);
+
+    // TODO - Remove hard coded values
+    const dateString = "June 30, 2020";
+
+    // TODO - Remove mock counts and replace with API call for counts
+    const mockCounts = {
+        all: 123123123,
+        contracts: 2856942,
+        idvs: 65718,
+        grants: 262180,
+        direct_payments: 3173522,
+        loans: 955562,
+        insurance: 123,
+        otherOption: 123123,
+        otherOption1: 123123123,
+        otherOption2: 123123123123
+    };
+
+    useEffect(() => {
+        const params = {
+            filter: {
+                def_codes: defCodes.map((defc) => defc.code)
+                // TODO - add award type codes
```

Comment:
We could go ahead and do this here since you've implemented the tabs, but I'm also fine to add it in the PR for the table itself.

Reply:
Agreed, but I'll add it in  the PR for the table because I had already done it for the table container.